proto/obb/session.proto

```diff
@@ -57,67 +55,107 @@ message InvalidSessionReason {
 enum SessionStatus {
   // Not a valid value. Used for blocking 0 as invalid state.
   SESSION_STATUS_UNSPECIFIED = 0;
+
+  // The session is valid and can still be used.
   SESSION_STATUS_VALID = 1;
+
+  // Session is not valid anymore.
   SESSION_STATUS_INVALID = 2;
 }
 
-message SessionRequest {
-  oneof data {
-    SessionCreate create = 1;
-    StreamKeepalive keepalive = 2;
-    SessionInfo info = 3;
-  }
+message SessionCreateRequest {
+  // List of desired selections to include in the new session.
+  // Format: "<event_id>/<market_id>/<outcome_id>?<market_specifier>"
+  // e.g. “od:match:1234/1/1?map=1&way=two”
+  repeated string selection_ids = 2;
 }
 
-message SessionResponse {
-  oneof data {
-    SessionCreateState create = 1;
-    StreamKeepalive keepalive = 2;
-    SessionInfoState info = 3;
+message SessionCreateResponse {
+  message CreatedSession {
```

Comment:
Nitpicking: I would rename `CreatedSession` -> `SessionCreated` and `RejectedSession` -> `SessionRejected`. Why? it fits better to property name  `oneof status`.

Reply:
updated